.github/workflows/build-packages.yml

```
@@ -243,3 +257,202 @@ jobs:
           mkdir -m 755 -p "slsa/${PRODUCT}/${VERSION}/"
           mv ${{steps.download-provenance.outputs.download-path}}/*.jsonl "slsa/${PRODUCT}/${VERSION}"
           rsync -4rlptD slsa/* "$RSYNCTARGET"
+
+  upload-packages:
+    needs:  [prepare, build, provenance-src, provenance-pkgs]
+    continue-on-error: true
+    name: Upload packages
+    runs-on: ubuntu-22.04
```

For consistency's sake, perhaps:
```suggestion
    runs-on: ubuntu-24.04
```